playwright/tests/fixture.ts

```diff
@@ -95,6 +94,7 @@ export const test = base.extend<{
       timeout: 20000,
     })
     await page.locator('.v-app-bar__nav-icon').click()
+    await expect(page.locator('#openc3-nav-drawer')).toBeHidden()
```

Comment:
I think this will also help because we're waiting for the drawer hide animation which gives everything a little more time to render.